recipes/libuvc/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake ${CMAKE_ARGS} .. \
+      -G "Ninja" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
```

Comment:
All of these can go as they are redundant with the `CMAKE_ARGS` we pass in through e.g. https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/6290f55b73a887086b0efe0cdb32553646696a8a/recipe/activate-gcc.sh#L137-L149

```suggestion
```